gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -129,13 +129,14 @@ nnstreamer_filter_find (const char *name)
 
 /**
  * @brief Parse the string of model
- * @param[out] prop Struct containing the properties of the object
  * @param[in] model_files the prediction model paths
+ * @param[out] model_file the parsed model_file
+ * @param[out] model_file_sub the parsed sub model_file
  * @return number of parsed model path
  */
 static guint
-gst_tensor_filter_parse_modelpaths_string (GstTensorFilterProperties * prop,
-    const gchar * model_files)
+gst_tensor_filter_parse_modelpaths_string (const gchar * model_files,
+    gchar ** model_file, gchar ** model_file_sub)
```

Comment:
By using "reload" and "setProperties", you won't need to modify such.

Reply:
Got it.